website/docs/sql-function-glossary/sql-aggregate-functions/sql-min.md

```diff
@@ -0,0 +1,58 @@
+---
+id: sql-min 
+title: SQL MIN 
+description: The MIN aggregate function allows you to compute the minimum value from a column or across a set of rows for a column.
+slug: /sql-function-glossary/sql-min
+---
+
+<head>
+    <title>Working with SQL MIN</title>
+</head>
+
+SQL MIN, MAX, SUM… the aggregate functions that you’ll live and die by as an analytics practitioner. In this post, we’re going to unpack the SQL MIN function, how to use it, and why it's valuable in data work.
```

Comment:
Not sure we need the additional space after the ... here

Reply:
we do not!

Reply:
Really? I've always added a space after. I think you can write it either way and it's still grammatically correct. Either way, I'll remove the space